Setup/UpgradeData.php

```diff
@@ -571,6 +571,41 @@ public function upgrade(ModuleDataSetupInterface $setup, ModuleContextInterface
             }
         }
 
+        if (version_compare($context->getVersion(), '4.2.0', '<=')) {
```

Comment:
Dit is misschien niet zo belangrijk, maar krijgen we hier ook niet weer dat de migratie 2 keer uitgevoerd gaat worden? Ik weet bijna zeker van wel, alleen ik denk dat deze migratie sowieso geen probleem is om meerdere keren uit te voeren toch? (Mijn magento kennis schiet hier te kort nog)

Reply:
hi ja maar ik denk dat het nog een keer uitgevoerd wordt als je naar de versie daarna gaat, net zoals we toen bij wc hadden geconstateert